README.md

```diff
@@ -1,2 +1,351 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+## get user
+
+- **Success Response:**
+  - **Code:** 200
+  - **Content:**
+
+```json
+{
+  "message": "Successfully retrieved user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+- **Code:** 500 SERVER ERROR
+- **Content:** `{"message": "Unexpected error", "data": [] }`
```

Comment:
'data' is not an array in the success route, so why is it an array in the failure route?

Reply:
done